GPU/Directx9/StateMappingDX9.cpp

```diff
@@ -140,6 +139,18 @@ void DrawEngineDX9::ApplyDrawState(int prim) {
 				ApplyFramebufferRead(&fboTexNeedsBind_);
 				// The shader takes over the responsibility for blending, so recompute.
 				ApplyStencilReplaceAndLogicOpIgnoreBlend(blendState.replaceAlphaWithStencil, blendState);
+
+				if (fboTexNeedsBind_) {
+					// Note that this is positions, not UVs, that we need the copy from.
+					framebufferManager_->BindFramebufferAsColorTexture(1, framebufferManager_->GetCurrentRenderVFB(), BINDFBCOLOR_MAY_COPY);
+					// If we are rendering at a higher resolution, linear is probably best for the dest color.
+					device_->SetSamplerState(1, D3DSAMP_MAGFILTER, D3DTEXF_LINEAR);
+					device_->SetSamplerState(1, D3DSAMP_MINFILTER, D3DTEXF_LINEAR);
+					fboTexBound_ = true;
+					fboTexNeedsBind_ = false;
+					gstate_c.Dirty(DIRTY_BLEND_STATE);
```

Comment:
Does this actually do anything?  It's already inside `if (gstate_c.IsDirty(DIRTY_BLEND_STATE)) {`, and that'll get cleared right after this either way...

-[Unknown]

Reply:
Eh, yeah, guess it doesn't heh. Some more cleanup to do...